### PR TITLE
Tell users that there is no help in mobile version, allow to switch to desktop

### LINKED ISSUE #4821: Mobile help view error 500
Going to the help page in mobile view produces a 500

```
Started GET "/help" for 213.245.69.149 at 2014-03-03 21:47:31 +0100
Processing by HelpController#faq as HTML
Completed 500 Internal Server Error in 6.5ms

ActionView::MissingTemplate (Missing template help/faq, application/faq with {:locale=>[:fr, :en], :formats=>[:mobile], :handlers=>[:erb, :builder, :markerb, :haml]}. Searched in:
  * "/home/diaspora/diaspora/app/views"
  * "/home/diaspora/.rbenv/versions/2.0.0-p195/lib/ruby/gems/2.0.0/gems/rails_admin-0.4.9/app/views"
  * "/home/diaspora/.rbenv/versions/2.0.0-p195/lib/ruby/gems/2.0.0/gems/kaminari-0.15.1/app/views"
  * "/home/diaspora/.rbenv/versions/2.0.0-p195/lib/ruby/gems/2.0.0/gems/devise-3.2.2/app/views"
):
  lib/rack/chrome_frame.rb:39:in `call'
  lib/unicorn_killer.rb:35:in `call'
```

Steps to reproduce:
Go to /help
Click on "toggle mobile" in the footer
##

### PR DESCRIPTION
**Pull request by Flaburgan**
Fixes #4821

Here is how it looks:

image

### CODE REVIEW

File: app/views/help/faq.mobile.haml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+%h1= t("_help")
+
+%p= t("help.not_available")
+%p= link_to(t("layouts.application.switch_to_standard_mode"), toggle_mobile_path)
```
Comment (diaspora-code-review) [inline app/views/help/faq.mobile.haml:4]: Files should end with a trailing newline

**Review by SuperTux88 — CHANGES_REQUESTED** [review]
First, the tests now fail because the page now "exists" on mobile.

Also, I would use the same design as is already used on `/admin/pods`, so we don't have two different ways of showing the same:
image

File: app/views/admins/pods.mobile.haml [hunk 2]
```diff
@@ -2,4 +2,4 @@
   = t(".pod_network")
 
 .alert.alert-warning
-  != t(".pod_desktop_view", desktop_link: link_to(t(".pod_desktop_link"), toggle_mobile_path))
+  != t("error_messages.mobile_not_available", desktop_link: link_to(t("error_messages.desktop_view_link"), toggle_mobile_path))
```
Comment (diaspora-code-review) [inline app/views/admins/pods.mobile.haml:5]: Line is too long. [127/120]

File: app/views/help/faq.mobile.haml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+%h1= t("_help")
+
+.alert.alert-warning
+  != t("error_messages.mobile_not_available", desktop_link: link_to(t("error_messages.desktop_view_link"), toggle_mobile_path))
```
Comment (diaspora-code-review) [inline app/views/help/faq.mobile.haml:4]: Line is too long. [127/120]

**Comment by Flaburgan** [comment]
@SuperTux88 I changed the translations keys so we can reuse them, the text actually is generic. Can you or @jhass do the action needed so it doesn't have to be re-translated?

**Review by SuperTux88 — APPROVED** [review]
LGTM :cookie:

**Comment by SuperTux88** [comment]
Merged and translations updated.

**Pull request merged**